configs/components/rubygem-ffi.rb

```diff
@@ -31,21 +31,23 @@
     @component.install = []
 
     if platform.architecture == "x64"
+      # NOTE: make sure to verify the shas for the x64-mingw32 gem!
```

Comment:
Do we need to mirror those versions of the gem separately, or does artifactory pick them up?

Reply:
Ah ffi contains native extensions, so `x64-mingw32` and `x86-mingw32` contain precompiled binaries for 64 and 32-bit Windows OS. It's a convenience thing, because most Windows users don't have the [devkit](https://rubyinstaller.org/downloads/) installed (and it's kind of a PITA).